src/sass/layout/_footer.scss

```diff
@@ -35,64 +57,115 @@
     margin-right: 136px;
   }
 }
-.footer__nav {
+.footer-nav__list {
   margin-top: 20px;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
   flex-direction: column;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
   align-items: center;
   text-align: center;
 }
 
-.footer__nav-item:not(:last-child) {
+.footer-nav__item:not(:last-child) {
   margin-bottom: 8px;
 }
-.footer__nav-link {
+.footer-nav__link {
   font-weight: 400;
   font-size: 16px;
   line-height: 1.25;
   letter-spacing: 0.01em;
   color: $main-text-color;
 }
-.footer__nav-link:hover,
-.footer__nav-link:focus {
+
+.footer-nav__link:hover,
+.footer-nav__link:focus {
   color: $accent-color;
 }
-
-.footer__social {
-  margin-top: 30px;
+.footer-social__list {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
   flex-direction: column;
+}
+.footer-social__list {
```

Comment:
Looks like this class styles are doubled - see line 87.
Please either explain the doubling reason, or remove it.

Reply:
deleted duplicate entry